pages/members/[id]/index.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import { useRouter } from 'next/router';
 import Profile from 'components/member-profile';
-
+import Layout from 'components/layout';
 const MemberProfile = () => {
```

Comment:
I guess arrow function will be always rebuilt, try using normal function. @ankushdharkar any comments.

Reply:
We are using Arrow functions in other files too,
And I did not work on the function, I have only added the layout component.
If it is necessary we can always make new issue and work on it.